Cargo.toml

```diff
@@ -8,7 +8,7 @@ homepage = "https://github.com/rust-lang/packed_simd"
 repository = "https://github.com/rust-lang/packed_simd"
 keywords = ["simd", "vector", "portability"]
 categories = ["hardware-support", "concurrency", "no-std", "data-structures"]
-license = "MIT/Apache-2.0"
+license = "MIT or Apache-2.0"
```

Comment:
The SPDX License expression standard has the `OR` operator in uppercase.
https://github.com/spdx/spdx-spec/blob/v2.2/chapters/appendix-IV-SPDX-license-expressions.md#case-sensitivity-